### ISSUE TITLE
One last step

### ISSUE THREAD

**Issue by Inginn** [andy-portmen/native-client#27]
I always get the One last step window pop up, this was no issue for the first months but closing all these windows is getting irritating, any way to turn this off.

**Comment by andy-portmen** [comment]
So the extension works normally and you still get this page? It should only appear if there is no native client or when the external application exited with an unsuccessful code. 

What is the extension name you are using and what are the arguments in the options page?

**Comment by Inginn** [comment]
yes the program works fine allways, just allways pops upp the "one last step" page
settings:

display name: MPV
Executable name: C:\Program Files\mpv\mpv.exe
Arguments [HREF]


toolbar button: off
Contextmenu: on
			on linked context

no icon file chosen

**Comment by andy-portmen** [comment]
I am not able to reproduce [screen.zip](https://github.com/andy-portmen/native-client/files/1224971/screen.zip)

**Comment by Chuunie** [comment]
Hi. I too am having this issue. Pretty much exactly the same setup as Inginn. To be more specific this occurs when the video in MPV has finished playing and MPV closes itself or when MPV is closed manually. It also does not occur every time, it appears to occur randomly, but frequently enough that it is a little bit of a nuisance. Let me know if you would like a screen recording.

**Comment by andy-portmen** [comment]
Go to the extension manager and turn the develop mode on. Now open the "background page" beneath the extension. When the "Open Last Step" page opens, there should be an error appearing on this page. If so let me know the error.

**Comment by Chuunie** [comment]
I don't see an error showing up in the console on that page after it occurs.

**Comment by Chuunie** [comment]
After trying to make it occur on purpose I notice the video needs to be open for a while or playing for a while for it to happen as I haven't seen it happen on videos I open and close within a few seconds while I've been testing just now, although that could be a coincidence.

**Comment by andy-portmen** [comment]
If you have a sample that causes this, please share the link. What extension are you using? External Application Button or Open in MPV?

**Comment by Chuunie** [comment]
I am using External Application Button on the latest stable version of Vivaldi on Windows 10. I've seen this occur on various youtube videos and I think twitch VODs too. For the sake of testing, just now I chose [this](https://www.youtube.com/watch?v=5iXzqN8-34E) video from my youtube feed. I right clicked it in my feed and opened it with MPV from my context menu, I then watched it all the way though and allowed MPV to automatically close itself, upon closing itself the "One last step" tab opened in my browser.
